packages/trace-viewer/src/sw/snapshotRenderer.ts

```diff
@@ -311,7 +311,17 @@ function snapshotScript(viewport: ViewportSize, ...targetIds: (string | undefine
           (element as HTMLElement).showPopover();
         } catch {
         }
-        element.removeAttribute('__playwright_popover_open_');
+        element.removeAttribute('__playwright_dialog_open_');
```

Comment:
```suggestion
        element.removeAttribute('__playwright_popover_open_');
```
shouldn't it be like that?